cargo-spatial/src/local.rs

```diff
@@ -0,0 +1,53 @@
+use crate::config::Config;
+use crate::opt::*;
+use std::ffi::OsString;
+use std::path::*;
+use std::process;
+use tap::*;
+
+/// Prepares and launches a local deployment.
+///
+/// Before launching the deployment, this will first run code generation and build
+/// workers in the project. Assumes that the current working directory is the root
+/// directory of the project, i.e. the directory that has the `Spatial.toml` file.
+pub fn launch(_opt: &Opt, _local: &Local, launch: &LocalLaunch) {
+    let config = Config::load().expect("Failed to load configuration");
```

Comment:
`"Failed to load configuration"` -> `"Failed to load configuration in current working directory"`

Reply:
I moved the logic for loading the config file into `main()`, since it already has better logic setup for performing error handling. I also improved the wording of the errors in `Config::load()` according to you suggestion 👍